### ISSUE TITLE
ImportError on Docker Image Build from Scratch

### ISSUE THREAD

**Issue by GvnCampbell** [digitalec/deemon#46]
I'm trying to build the docker image from scratch using the Dockerfile provide.  The build goes fine but when I try to run deemon in the docker I get the following error.  It looks like something my have broken in one of the dependencies.

The prebuilt docker image though works.

```
Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/app/deemon/__main__.py", line 1, in <module>
    from deemon import cli
  File "/app/deemon/cli.py", line 11, in <module>
    from deemon.cmd import download, rollback, backup, extra
  File "/app/deemon/cmd/download.py", line 12, in <module>
    from deemon.core import dmi, db
  File "/app/deemon/core/dmi.py", line 12, in <module>
    from deezer.gw import GWAPIError, LyricsStatus
ImportError: cannot import name 'LyricsStatus' from 'deezer.gw' (/usr/local/lib/python3.8/dist-packages/deezer/gw.py)
```

**Comment by RemixDev** [comment]
deemix 3.6 released along side with deezer-py 1.3
these new versions bring breaking changes with older versions of the app (for example LyricsStatus is now in utils instead of in gw)

**Comment by digitalec** [comment]
Fixed in 2.8.1. Updated requirements.txt to pull only compatible versions so this should prevent the docker image from breaking in the future as well.
